src/map/utils/battleutils.cpp

```diff
@@ -4363,6 +4362,62 @@ namespace battleutils
         return reductionPercent;
     }
 
+    void BindBreakCheck(CBattleEntity* PAttacker, CBattleEntity* PDefender)
+    {
+        if (PDefender->StatusEffectContainer->HasStatusEffect(EFFECT_BIND))
+        {
+            uint16 BindBreakChance = 0; // 0-1000 (100.0%) scale. Maybe change to a float later..
+            uint16 LvDiffByExp = charutils::GetRealExp(PAttacker->GetMLevel(), PDefender->GetMLevel()); // This is temp.
+
+            // Todo: replace with an actual calculated value based on level difference. Not it, Bro!
+            // This entire block of conditionals should not exist. Lv diff really shouldn't be handled at the exp check level either.
+            // It might not even be in sync with the check values the entire way from lv 1 to lv 99 for all we know.
+            if ((PAttacker->objtype == TYPE_PC || PAttacker->objtype == TYPE_PET) && PDefender->objtype == TYPE_MOB)

```

Comment:
I'd edit to account for the day pvp returns, but that whole conditional branch will likely be changed to something that doesn't need object types anyway (used the /check because we have real data)